portal-impl/test/unit/com/liferay/portal/servlet/filters/strip/StripFilterTest.java

```diff
@@ -213,6 +213,47 @@ public void testProcessCSS() throws Exception {
 		Assert.assertEquals(code.length() + 34, charBuffer.position());
 	}
 
+	@Test
+	public void testProcessInput() throws Exception {
+		StripFilter stripFilter = new StripFilter();
+
+		CharBuffer charBuffer = CharBuffer.wrap("<input ><br />");
```

Comment:
@rolandpakai Change you input here to "<input ><br>dddd</br>", and run you test, it will pass too. Which means you removed a line "<br>dddd</br>" here. This is not what StripFilter should do.

Reply:
Hi @tinatian 

thanks for checking the unit test.

In case of `<input ><br>dddd</br>` the  `Assert.assertEquals(8, charBuffer.position());` test will pass too,
which means the `<br>dddd</br>` is not removed.

I created the following testcase and it pass too, which means the `<br>dddd</br>` is not removed.

```
stripFilter.strip(
     null, null, CharBuffer.wrap("<input ><br>dddd</br>"), stringWriter);
Assert.assertEquals("<input ><br>dddd</br>", stringWriter.toString());
```
I sent the PR again here: https://github.com/liferay-core-infra/liferay-portal/pull/889

Thansk, Roland

Reply:
@rolandpakai I did not see the code you list above in your new pull, please verify. 
Also test processInput is confusing, please test strip directly to avoid the confused test logic.

Tina.

Reply:
@tinatian 
I didnt add that test case because it tests the `strip `method and not the `processInput`

I added test for the `strip `method

Check the new PR here: https://github.com/liferay-core-infra/liferay-portal/pull/889